db/migrate/20220125182817_remove_user_id_from_buildings.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveUserIdFromBuildings < ActiveRecord::Migration[6.1]
+  def change
+    remove_column :buildings, :user_id, :integer
+  end
+end
```

Comment:
Good intention, but I think we should maybe just remove these migrations and the ones which add the columns? In my opinion, as long as the migration didn't enter dev, it doesn't matter if you have to drop the database